cleanlab/classification.py

```diff
@@ -623,26 +705,62 @@ def find_label_issues(
 
         if self.verbose:
             print("Using predicted probabilities to identify label issues ...")
-        # Get the boolean mask of the label issues
         self.label_issues_mask = filter.find_label_issues(
             labels,
             pred_probs,
             **self.find_label_issues_kwargs,
         )
+        label_quality_scores = get_label_quality_scores(
+            labels, pred_probs, **self.label_quality_scores_kwargs
+        )
+        self.label_issues_df = pd.DataFrame(
+            {"is_label_issue": self.label_issues_mask, "label_quality": label_quality_scores}
+        )
+        if pred_probs is not None:
```

Comment:
is this supposed to be `if self.label_issues_df` is not None`?

Reply:
if pred_probs was None here, wouldn't the previous lines fail?

Reply:
how do you know that self.label_issues_df has been created yet if the workflow is
```python
# Find label issues in a dataset in one line of code.
label_issues_df = CleanLearning(model).find_label_issues(labels, pred_probs)
```